package/network/ipv6/ds-lite/Makefile

```diff
@@ -17,13 +17,13 @@ define Package/ds-lite
   SECTION:=net
   CATEGORY:=Network
   DEPENDS:=@IPV6 +kmod-ip6-tunnel +resolveip
-  TITLE:=Dual-Stack Lite (DS-Lite) configuration support
-  MAINTAINER:=Steven Barth <steven@midlink.org>
+  TITLE:=IPv4 over IPv6 (RFC2473/DS-Lite) configuration support
```

Comment:
As @systemcrash pointed out, this is a bit ambiguous.
You can use 'and' instead of '/' here.
Same for the description a few lines below.